core/templates/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.component.html

```diff
@@ -98,7 +99,7 @@ <h3>Load dummy new structures data (only admins)</h3>
 
   <mat-card class="oppia-page-card oppia-long-text oppia-long-text-card">
     <div class="container ml-0">
-      <h3>Generate dummy skill with questions (only admins)</h3>
+      <h3>Generate dummy skill with questions (only curriculum admins)</h3>
```

Comment:
Here and below, is it worth giving folks instructions on how to become a curriculum admin here in small grey text? ("To get curriculum admin access, go to the Roles tab and assign this role to your username.")

Reply:
It is done.

Reply:
@seanlip I made all the changes suggested by you